ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py

```diff
@@ -92,29 +92,6 @@ def test_numpy_ndarray_property_ndim(
     ivy.utils.assertions.check_equal(x.ndim, data[0].ndim, as_array=False)
 
 
-@given(
```

Comment:
why have you removed this test?

Reply:
Hey Sam -  Thanks a lot for your feedback, I'm not sure how I came to remove it accidentally! rolling it back :)